streams/src/test/java/org/apache/kafka/streams/processor/internals/GlobalStreamThreadTest.java

```diff
@@ -189,16 +199,6 @@ public void shouldCloseStateStoresOnClose() throws Exception {
         assertFalse(globalStore.isOpen());
     }
 
-    @Test
-    public void shouldTransitionToDeadOnClose() throws Exception {
```

Comment:
Why remove this test?

Reply:
Because test `shouldStopRunningWhenClosedByUser()` above is exactly the same test.

Reply:
Thx!